packages/grafana-runtime/src/services/live.ts

```diff
@@ -32,6 +32,7 @@ export interface StreamingFrameOptions {
   maxLength: number; // 1000
   maxDelta: number; // how long to keep things
   action: StreamingFrameAction; // default will append
+  decodeLabelsOnly?: boolean;
```

Comment:
Maybe better:
```suggestion
  targetFrameType?: string;
```

And we add a frame type for "timeseries-labeled" (default is [timeseries-wide](https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/frame_type.go#L21))

Reply:
POC at https://github.com/grafana/grafana-plugin-sdk-go/pull/483